package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bridge-in-tech-web",
   "version": "0.1.0",
   "private": true,
-  "homepage": "http://localhost:3000/",
+  "homepage": "https://anitab-org.github.io/bridge-in-tech-web/",
```

Comment:
@jalajcodes and @shades-7, I'd rather we just comment out the localhost option instead of deleting it totally. If we comment it out, people who wants to use the localhost can still use it if they want to. I think we're still going to need/use the localhost while we're changing the backend CORS to include the new deployed url.

Reply:
@mtreacy002 Makes sense.
@shades-7 please add the line back and comment it out.

Reply:
The JSON is data only, and if I include a comment, then it will be data too.
{
   "_comment": "comment text goes here...",
}
so should I do this?